bin/infection

```diff
@@ -38,6 +38,12 @@ $autoloaderInWorkingDirectory = getcwd() . '/vendor/autoload.php';
 
 if (is_file($autoloaderInWorkingDirectory)) {
     require_once $autoloaderInWorkingDirectory;
+} else {
+    $vendorDir = exec('composer config vendor-dir');
```

Comment:
relying on an `exec()` and `getcwd()` is quite tricky here... At the very least I would use it as a last restart after we try to load the other files. Plus as @sanmai noted you need to gracefully fail when it fails.